Userland/Services/WindowServer/WMClientConnection.cpp

```diff
@@ -170,5 +170,4 @@ void WMClientConnection::handle(const Messages::WindowManagerServer::SetWindowTa
     auto& window = *(*it).value;
     window.set_taskbar_rect(message.rect());
 }
-
```

Comment:
Please don't make whitespace changes in files that are otherwise not changed, especially not like this where there actually should be an empty line :)